ci/infrastructure/scripts/infrastructure.sh

```
@@ -234,9 +234,11 @@ defaultSettings() {
   if [ -z "$VS_CONTAINER_NAME" ]&&[ "$VS_BRANCH_NAME" != "branch-not-found" ]; then
     VS_CONTAINER_NAME=$(dirname $JOB_NAME | sed -e "s/\//_/g")"_"$(basename $VS_BRANCH_NAME)
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)